k8s/prod/cluster-00/sscs/sscs-case-loader.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     path: stable/sscs-case-loader
   values:
     job:
-      schedule: "0 0/1 * * *"
+      schedule: "0 2/1 * * *"
```

Comment:
“At minute 0 past every hour from 2 through 23.”  Not too sure this is what we are aiming for. We need to put a cap on the retries so it stops by start of business.